pkg/static/login.js

```diff
@@ -11,6 +11,22 @@
         console.warn(String(ex));
     }
 
+    /* Dark mode */
+    const theme = localStorage.getItem('shell:style');
+    if ((window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches && theme === "auto") || theme === "dark") {
+        document.documentElement.classList.add('pf-theme-dark');
+    } else {
+        document.documentElement.classList.remove('pf-theme-dark');
+    }
+
+    window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', event => {
+        if ((event.matches && theme === "auto") || theme === "dark") {
+            document.documentElement.classList.add('pf-theme-dark');
+        } else {
+            document.documentElement.classList.remove('pf-theme-dark');
+        }
+    });
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17914-20221201-104014-14c78ccb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)